.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: python
+python:
+- '2.7'
+- '3.4'
+- '3.5'
+- '3.6'
+cache: pip
+before_install:
+- npm install npm@latest -g
+install:
+- pip install tox-travis
+- pip install bumpversion
+- npm install @semantic-release/exec
+script:
+- tox
+before_deploy:
+- sudo apt-get update
+- pip install -r requirements.txt
+- pip install -r requirements-dev.txt
+- pip install pypandoc
+- sudo apt-get install pandoc
+- pip install --editable .
+deploy:
+- provider: script
+  script: npx semantic-release
+  skip_cleanup: true
+  on:
+    python: '3.5'
+    branch: master
+- provider: pypi
+  user: ehdsouza
+  password:
+    secure: eMssjS1Airz5MrmcEDAnKYnTlMLuzinX8JFsh4TB0A4t7GCes2a/mlvcoazAv2QZMQUKQMC++bAXYGGgXtsSQsfOYnRtojW4pRphxOZrlXlWFgCLGz/eHioamNZVbH+/P/OYjbLy5ZKSzp3M3bZxYqHcmGTq0sBeo5A2Ilkzu5P+D865Jqtn52brwh/xYq4XWeJeQN/BSLwdtPEtZYQXhQupAteNDAo2ON0FVyt0amINrjA0Qw+0k35g36wti+CsKn5fhbaHf6jjN0oRj8J8wPZieXLAUMdxscfBjUXJ8AK8E4H/wPI30IsaMus+1r85ekiDKZC5iukO6uqMARFywjX/eTIDbWxwKImQUgvVa2+gncre7GvZrHIFxhIDDkssH1IfC1fcw3mNY4vCO9df/vHStp3Kvi27GvsYJnfsbXXj6fAhokHtsrnrbpXFET6LghGQx3HgOKrlCQ1EI0Fk92Qw7+g15jwMGlkwUE1qMS+PVOe7dQ4ikxJgw4CVrqU6/zaLFHV8LXFfjv166ZSWcOJHJXGgJ2TPuGiooL60+pWp/UYY00hZVlYX0Vlks6zN0h9zJd16kR3EwDrW0muqFnLC2xTuv5WsCjsOATtsTQzmBNEdCbGdOskgF4+0yPe7ps3kEs3UpAooKtYVh1OOJAUkl4vSMyu4Jg1ASyQWt00=
```

Comment:
Will this work?

Reply:
@germanattanasio followed these steps for encrypting key: https://docs.travis-ci.com/user/deployment/pypi/